src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/health-icon.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { HealthIcon } from '../enum/health-icon.enum';
+
+@Pipe({
+  name: 'healthIcon'
+})
+export class HealthIconPipe implements PipeTransform {
+  transform(value: string): string {
+    return Object.keys(HealthIcon).includes(value as HealthIcon) ? HealthIcon[value] : null;
```

Comment:
```suggestion
    return Object.keys(HealthIcon).includes(value as HealthIcon) ? HealthIcon[value] : '';
```
Returning null to the `[ngClass]` will give an error, maybe is better to return an empty string?

Reply:
done!